src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1572,6 +1580,8 @@ impl A320BrakingForce {
         self.right_braking_force = self.right_braking_force.max(0.).min(1.);
 
         self.correct_with_flaps_state(context);
+
+        self.apply_chocks_braking();
```

Comment:
I should have probably better named this "update_chocks_braking"

Reply:
@crocket63 I can change that.

Reply:
@crocket63 Changed to `update_chocks_braking`